modules/serverlesss-ossm-external-certs.adoc

```diff
@@ -0,0 +1,45 @@
+[id="serverlesss-ossm-external-certs_{context}"]
+= Creating a certificate to encrypt incoming external traffic
+
+The {ProductShortName} mTLS feature by default only secures traffic inside of the {ProductShortName} itself, between the ingress gateway and individual pods that have sidecars. To encrypt traffic as it flows into the {product-title} cluster, you must generate a certificate before you enable the {ServerlessProductName} and {ProductShortName} integration.
```

Comment:
s/The {ProductShortName} mTLS feature by default/By defult, the {ProductShortName} mTLS feature ?? 
Seems less awkward.